lib/webservice/club/club_webservice_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:io';
+
+import 'package:dio/dio.dart';
+import 'package:flutter_template/model/webservice/club/club.dart';
+import 'package:flutter_template/util/app_constants.dart';
+
+class WebserviceTest {
+  String baseUrl = 'https://www.strava.com/api/v3';
```

Comment:
Use retrofit & dependency injection instead of parsing & creating your down dio instance.